.github/ISSUE_TEMPLATE/software-request.md

```diff
@@ -1,6 +1,9 @@
 ---
 name: Software / Feature request
 about: Suggest an application or a feature
+title: ''
+labels: ''
+assignees: ''
```

Comment:
These 3 lines were automatically when creating a new template via Settings > General > Features > Set up templates.

One can set a default title, labels and assignees for each template, see: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-templates